src/main/scala/org/ergoplatform/nodeView/state/UtxoStateReader.scala

```diff
@@ -24,8 +24,7 @@ trait UtxoStateReader extends ErgoStateReader with TransactionValidation {
 
   val constants: StateConstants
 
-  private lazy val np = NodeParameters(keySize = 32, valueSize = None, labelSize = 32)
-  protected lazy val storage = new VersionedLDBAVLStorage(store, np)
+  protected lazy val storage = new VersionedLDBAVLStorage(store)
```

Comment:
Do we require protected methods to have ScalaDoc?

Reply:
isd but added as this class is very important one